From 1131fec553a15bc3c6006ee9f0943b5a6f9e999a Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Fri, 13 Feb 2026 01:20:10 -0700
Subject: [PATCH] 1.4.0-cmake-minimum

* Set policy_max which defines the maximum CMake policy. cmake_minimum_required
  sets two minimums: the real (lower) minimum and the (upper) minimum based on
  has been tested with. The defaults (CMake policies) will be set to the highest
  value possible between the two values.
   - c.f. https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index caae494..174fe1c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@
 # along with KLFitter. If not, see <http://www.gnu.org/licenses/>.
 
 # Set the minimum CMake version required to build the project.
-cmake_minimum_required( VERSION 3.1 )
+cmake_minimum_required(VERSION 3.1...4.2)
 
 # Silence some warnings on macOS with new CMake versions.
 if( NOT ${CMAKE_VERSION} VERSION_LESS 3.9 )
-- 
2.52.0

